src/content/popup/popup.ts

```diff
@@ -36,6 +36,7 @@ import { renderTabBar } from './tabs';
 import { renderWordEntries } from './words';
 
 import popupStyles from '../../../css/popup.css';
+import { onHorizontalSwipe } from './swipe';
```

Comment:
Nit: Can we move this up to line 34? i.e. before the import from `'./tabs'` to keep it in alphabetical order?

(Sorry, I haven't gotten around to defining proper linting rules for import order.)